crates/storage/src/job.rs

```diff
@@ -432,8 +432,40 @@ mod jobs {
     impl Job for DeactivateUserJob {
         const NAME: &'static str = "deactivate-user";
     }
+
+    /// Send account recovery emails
+    #[derive(Serialize, Deserialize, Debug, Clone)]
+    pub struct SendAccountRecoveryEmailJob {
```

Comment:
maybe make this plural (Email**s**) so it's a bit clearer it does multiple of them

Reply:
Done in 369d8b1b